cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/AbstractConnectOperator.java

```diff
@@ -652,29 +677,89 @@ private  Future<ConnectorStatusAndConditions> autoRestartFailedConnectorAndTasks
     }
 
     /**
-     * Connector or tasks failed should only restart at minute 0, 2, 6, 12, 20 and 30
+     * Restarts the connector and updates the auto-restart status.
+     *
+     * @param reconciliation                Reconciliation marker
+     * @param host                          Kafka Connect host
+     * @param apiClient                     Kafka Connect REST API client
+     * @param connectorName                 Name of the connector
+     * @param status                        The new/future status of the connector
+     * @param previousAutoRestartStatus     Previous auto-restart status used to generate the new status (increment the restart counter etc.)
+
+     * @return  Future with connector status and conditions which completes when the connector is restarted
+     */
+    private Future<ConnectorStatusAndConditions> autoRestartConnector(Reconciliation reconciliation, String host, KafkaConnectApi apiClient, String connectorName, ConnectorStatusAndConditions status, AutoRestartStatus previousAutoRestartStatus) {
+        LOGGER.infoCr(reconciliation, "Auto restarting connector {}", connectorName);
```

Comment:
Noting that this is an increase in logging level a previously this log was at debug level. I prefer having it at info level as I think it is useful for the user to know when strimzi has restarted a connector.

Reply:
That was intentional. I think it is important enough information to be logged at INFO which is what most users use.